.github/workflows/ci.yml

```diff
@@ -59,4 +59,3 @@ jobs:
           # The codecov token to upload report. Can be retrieved from your Codecov dashboard.
           # This should be set from `GitHub Repo > Settings > Secrets`.
           token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: true
```

Comment:
any reason for removing this?

Reply:
Yeah, the integration is not 100% perfect so this is giving trouble. I'll look more into this for open source projects. Ideally we don't need any token for open source repos.

Having this in place is kind of false alarm that the test cases didn't succeed.